README.md

```diff
@@ -1,24 +1,17 @@
-![Wallaroo Logo](wallaroo-logo.png)
+<p align="center"><a href="https://www.wallaroolabs.com/"><img src="wallaroo-logo.png" alt="WallarooLabs logo" width="400"/></a></p>
+<h2 align="center">Build and scale real-time data applications as easily as writing a script</h2>
+
 ---
+[![CircleCI](https://circleci.com/gh/WallarooLabs/wallaroo.svg?style=shield)](https://circleci.com/gh/WallarooLabs/wallaroo)
+[![GitHub license](https://img.shields.io/badge/license-apache%202-blue.svg)][wallaroo-community-license-readme]
+[![GitHub version](https://badge.fury.io/gh/WallarooLabs%2Fwallaroo.svg)](http://badge.fury.io/gh/WallarooLabs%2Fwallaroo)
```

Comment:
For this to work properly I need to add a webhook that points to `https://badge.fury.io/hooks/github`